Modules/Loadable/Segmentations/Widgets/qMRMLSegmentationConversionParametersWidget.cxx

```diff
@@ -361,11 +363,11 @@ void qMRMLSegmentationConversionParametersWidget::populateParametersTable()
       QLabel* textValueLabel = new QLabel(parameterValue, geometryWidget);
       geometryLayout->addWidget(textValueLabel);
 
-      QPushButton* setGeometryFromVolumeButton = new QPushButton(tr("Set geometry from volume"), geometryWidget);
-      setGeometryFromVolumeButton->setFixedWidth(160);
-      QObject::connect(setGeometryFromVolumeButton, SIGNAL(clicked()),
-                       this, SLOT(onSetReferenceImageGeometryFromVolumeClicked()));
-      geometryLayout->addWidget(setGeometryFromVolumeButton);
+      QPushButton* specifyGeometryButton = new QPushButton(tr("Specify geometry"), geometryWidget);
+      //setGeometryFromVolumeButton->setFixedWidth(160);
```

Comment:
if not needed then it should be probably deleted

Reply:
I left it there because it was not obvious what will be the proper solution. You hated the long widget after resizing it manually, but I'm not sure this is better.

Reply:
I'll remove this. I thought this is for the height issue, but it's not. Thanks